src/bioregistry/data/bioregistry.json

```diff
@@ -2864,9 +2864,24 @@
       "prefix": "ARK",
       "uri_format": "http://identifiers.org/ark/$1"
     },
+    "contact": {
+      "email": "jakkbl@gmail.com",
+      "github": "jkunze",
+      "name": "John Kunze",
+      "orcid": "0000-0001-7604-8041"
+    },
+    "contributor_extras": [
+      {
+        "email": "jakkbl@gmail.com",
+        "github": "jkunze",
+        "name": "John Kunze",
+        "orcid": "0000-0001-7604-8041"
+      }
+    ],
+    "example": "/53355/cl010066723",
     "fairsharing": {
       "abbreviation": "ARK Identifier",
-      "description": "Archival Resource Keys (ARKs) serve as persistent identifiers, or stable, trusted references for information objects. Among other things, they aim to be web addresses (URLs) that don’t return 404 Page Not Found errors. The ARK Alliance is an open global community supporting the ARK infrastructure on behalf of research and scholarship. End users, especially researchers, rely on ARKs for long term access to the global scientific and cultural record. Since 2001 some 8.2 billion ARKs have been created by over 750 organizations — libraries, data centers, archives, museums, publishers, government agencies, and vendors. ARKs are open, mainstream, non-paywalled, decentralized persistent identifiers that researchers can begin to work with in a short time. They identify anything digital, physical, or abstract.",
+      "description": "Archival Resource Keys (ARKs) serve as persistent identifiers, or stable, trusted references for information objects. Among other things, they aim to be web addresses (URLs) that don’t return 404 Page Not Found errors. The ARK Alliance is an open global community supporting the ARK infrastructure on behalf of research and scholarship. End users, especially researchers, rely on ARKs for long term access to the global scientific and cultural record. Since 2001 some 8.2 billion ARKs have been created by over 1000 organizations — libraries, data centers, archives, museums, publishers, government agencies, and vendors. ARKs are open, mainstream, non-paywalled, decentralized persistent identifiers that researchers can begin to work with in a short time. They identify anything digital, physical, or abstract.",
```

Comment:
I think it would be better to update this text in FAIRsharing - are you the  owner of the ARK record there (https://fairsharing.org/FAIRsharing.f928f1)? 

Otherwise, we can use the `"description"` key to override the description for the Bioregistry entry

Reply:
cc @allysonlister it appears the ARK record on FAIRsharing doesn't have a maintainer. Perhaps you can chat with @jkunze about having him take it over as maintainer and make some updates.

Reply:
In the meantime I will revert this and merge the PR

Reply:
Hi all!

Thanks @cthoyt for pinging us, and thanks for claiming the record @jkunze - I have approved your ownership, just let me know if you have any questions about updating records - our documentation is at https://fairsharing.gitbook.io/fairsharing/

Reply:
Thank you, Allyson. I updated the ARK entry at fairsharing.